rxjava3/src/main/java/io/micronaut/rxjava3/converters/RxJava2ToRxJava3ConveterRegistrar.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2017-2020 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.rxjava3.converters;
+
+import hu.akarnokd.rxjava3.bridge.RxJavaBridge;
+import io.micronaut.context.annotation.BootstrapContextCompatible;
+import io.micronaut.context.annotation.Requires;
+import io.micronaut.core.convert.ConversionService;
+import io.micronaut.core.convert.TypeConverterRegistrar;
+import io.reactivex.rxjava3.core.Flowable;
+import io.reactivex.rxjava3.core.Maybe;
+import io.reactivex.rxjava3.core.Single;
+import io.reactivex.rxjava3.core.Completable;
+import jakarta.inject.Singleton;
+
+/**
+ * Converters between RxJava 2 and  RxJava 3.
+ *
+ * @author Sergio del Amo
+ * @since 2.0.0
+ */
+@Singleton
+@Requires(classes = {
+        io.reactivex.Flowable.class,
+        io.reactivex.Maybe.class,
+        io.reactivex.Single.class,
+        io.reactivex.Completable.class,
+        io.reactivex.Observable.class})
+@BootstrapContextCompatible
+public class RxJava2ToRxJava3ConveterRegistrar implements TypeConverterRegistrar {
```

Comment:
Is this class to allow to use RxJava2 and 3 at the same time? Is it recommended to do it?

Reply:
No, I don't think it is recommended. I think we should encourage users to use a single reactive library.